packages/plans-grid-next/src/components/shared/action-button/style.scss

```diff
@@ -0,0 +1,26 @@
+.plans-grid-next__action-button {
+	&-multi {
```

Comment:
I suggest avoiding this form of string concatenation for class names. It makes it impossible to search for the full name. Debugging experience is quite bad with these.

Reply:
Sure, updated: ec3069e